app/bundles/CampaignBundle/Controller/EventController.php

```diff
@@ -120,6 +120,7 @@ public function newAction()
                 $formThemes[] = $event['settings']['formTheme'];
             }
 
+            // @todo
```

Comment:
Having an issue here with converting the php template to twig. Need to come back to resolve later.

When using the twig template and creating a new campaign and with the builder launched, when you select "Decision", "Action", etc. it will throw a 500 error and fail to load.